py/picca/delta_extraction/astronomical_objects/desi_forest.py

```diff
@@ -22,20 +22,16 @@ class DesiForest(Forest):
     blinding: str (from Forest)
     Name of the blinding strategy used
 
-    lambda_grid: array of float or None (from Forest)
-    Common grid in lambda_ based on the specified minimum and maximum
-    wavelengths, and delta_lambda.
-
-    lambda_rest_frame_grid: array of float or None (from Forest)
-    Same as lambda_grid but for rest-frame wavelengths
-
     log_lambda_grid: array of float or None (from Forest)
     Common grid in log_lambda based on the specified minimum and maximum
     wavelengths, and delta_log_lambda.
```

Comment:
this doc string makes it look like log_lambda_grid depends on delta_log_lambda, but when using linear pixels (that should be the default soon) it is delta_lambda instead that is used.

Reply:
I added a commit changing the docstrings. I updated this description in the Forest docstring. I decided to simplify the docstrings to only contain the elements defined there and leave the inherited attributes on the parent class. This is to simplify maintenance.